src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -61,51 +82,45 @@ const BenefitFormUI: React.FC = () => {
-
-          return isRequired;
-        });
-        const allSchema = {
-          ...applicationSchema,
-          required,
-          properties,
-        };
-
-        setFormSchema(allSchema);
+      if (isRequired !== undefined) {
+        delete properties[key].required;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)